FirebaseRemoteConfig/Sources/RCNConfigContent.m

```diff
@@ -284,6 +306,14 @@ - (void)loadConfigFromMainTable {
                    self->_defaultConfig = [defaultConfig mutableCopy];
                    dispatch_semaphore_signal(self->_configLoadFromDBSemaphore);
                  }];
+
+  [_DBManager loadPersonalizationWithCompletionHandler:^(BOOL success, NSDictionary *fetchedConfig,
```

Comment:
rename to `fetchedPersonalization` and `activePersonalization`?

Reply:
Done.